common/djangoapps/edxmako/paths.py

```diff
@@ -51,13 +52,14 @@ def get_template(self, uri):
         """
         Overridden method which will hand-off the template lookup to the microsite subsystem
         """
-        microsite_template = microsite.get_template(uri)
+        if microsite.is_request_in_microsite():
```

Comment:
These microsite references should be routed through the theming app -- the goal of this work is two-fold: to support multiple sites with comprehensive theming and eliminate the existing microsites implementation.